alfred-telemetry-solr/alfred-telemetry-solr-common/src/main/java/eu/xenit/alfred/telemetry/solr/util/Util.java

```diff
@@ -31,10 +31,10 @@ public static boolean isEnabled(String env) {
             return Boolean.parseBoolean(System.getenv(env));
         }
 
-        // everything is enabled by default except graphite registry and tomcat metrics
+        // everything is enabled by default except graphite registry, tomcat and jetty metrics
```

Comment:
Instead of making this conditional larger and larger, can you instead keep a `Set<String>` of environment variables for which metrics are default disabled?

At class level:
```java
private static final Set<String> DEFAULT_DISABLED_CONFIGS;
static {
Set<String> configs = new HashSet<>();
configs.add("ALFRED_TELEMETRY_EXPORT_GRAPHITE_ENABLED");
configs.add("METRICS_TOMCAT_ENABLED");
configs.add("METRICS_JETTY_ENABLED");
DEFAULT_DISABLED_CONFIGS = Collections.unmodifiableSet(configs);
}
```

and then here:
```
DEFAULT_DISABLED_CONFIGS.contains(env)?false:true;
```

I think that makes the code a bit more readable and more easily extendable.